docs/man/docker-cp.1.md

```diff
@@ -11,8 +12,9 @@ CONTAINER:PATH HOSTPATH
 
 # DESCRIPTION
 Copy files/folders from a container's filesystem to the host
-path. Paths are relative to the root of the filesystem. Files
-can be copied from a running or stopped container.
+path running the command. Paths are relative to the root of
```

Comment:
Hi Doug --- (unless it starts with /) --- what does it refer to? "the path in the container" or "HOSTPATH" or something else.  (This isn't really a commentary on the issue...just wanting to learn myself)